Assets/Scripts/AdsAndIAP/CoinsAdReward.cs

```diff
@@ -0,0 +1,27 @@
+﻿/*
+ * Author(s): Grace Barrett-Snyder
+ * Description: An in-game reward for watching an ad that gives the player coins. 
+ */
+
+using k = PPGlobal;
+
+public class CoinsAdReward : AdReward
+{
+    PPGameController gameController;
+    int amount;
+
+    public CoinsAdReward(int amount = 100)
```

Comment:
Also confused as to why this constructor takes an `amount` argument if the `amount` class value is set based on a tuning value